### PR TITLE
Add password policy

### PR DESCRIPTION
**Pull request by Sneezry**
Snipaste_2021-07-08_14-55-49

### CODE REVIEW

**Comment by Sneezry** [comment]
@mymindstorm any idea with the test failure? It seems the test engine failed to connect to the browser.

File: _locales/en/messages.json [hunk 1]
```diff
@@ -390,6 +390,8 @@
     "backup_file_info": {
         "message": "Backup your data to a file."
     },
+    "password_policy_default_hint": {
+        "message": "The password doesn't meet the security requirements. Contact with your administrator."
```
Comment (mymindstorm) [inline _locales/en/messages.json]:
```suggestion
        "message": "Your password does not meet your organization's security requirements. Contact your administrator for more information."
    },
```

This is why the tests are failing. The json is invalid, causing the extension to not load. Therefore, the test runner is unable to communicate with view/test.html

**Review by mymindstorm — APPROVED** [review]
Testing worked perfectly. I think an example could be added in `schema.json`, just to clarify that no slashes are needed (e.g. `/[A-Z]+/` should be `[A-Z]+`).

**Comment by Sneezry** [comment]
> Testing worked perfectly. I think an example could be added in `schema.json`, just to clarify that no slashes are needed (e.g. `/[A-Z]+/` should be `[A-Z]+`).

Updated.

**Pull request merged**